<launch>
 <group ns="realsense">
    <param name="global_time" type="bool" value="true" />
    <param name="sync_size" type="int" value="30" />
    <param name="enable_emitter" type="bool" value="false" />

    <param name="frame_rate" type="int" value="30" />
    <param name="stereo_format" type="str" value="Y8" />
    <param name="depth_format" type="str" value="Z16" />
    <param name="rgb_format" type="str" value="RGB8" />
    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="exposure" type="double" value="10000" />
    <param name="auto_exposure" type="bool" value="true" />
    <param name="auto_white_balance" type="bool" value="true" />
  </group>

  <node pkg="myrealsense"
      type="intel_d435i"
      name="intel_d435i"
      required="true"
      output="screen">
  </node>
</launch>
